api/scripts/update-codegen.sh

```diff
@@ -30,6 +30,9 @@ function source::settings {
   SCRIPT_ROOT="${SCRIPT_DIR}/.."
   CODEGEN_PKG="$(go env GOMODCACHE)/$(go list -f '{{.Path}}@{{.Version}}' -m k8s.io/code-generator)"
   MODULE="github.com/deckhouse/virtualization/api"
+  PREFIX_GROUP="virtualization.deckhouse.io_"
+  # TODO: legacy, delete after all crds are auto-generated
+  ALLOWED_RESOURCE_GEN_CRD=("VirtualMachineClass ExampleKind1 ExampleKind2")
```

Comment:
New code is not legacy ;)

```suggestion
  # TODO: Temporary filter until all CRDs become auto-generated.
  ALLOWED_RESOURCE_GEN_CRD=("VirtualMachineClass ExampleKind1 ExampleKind2")
```